xml/article_nfs_storage.xml

```diff
@@ -450,255 +454,307 @@ include /etc/drbd.d/*.res;</screen>
+         <literal>/srv/nfs/share</literal>. </para>
     </listitem>
     <listitem>
-        <para>Into this export directory, the cluster will mount
-            <literal>ext3</literal> file systems from the DRBD device
-         <filename>/dev/drbd0</filename>.
-         This DRBD device sits on top of an LVM logical volume with the name
-         <literal>nfs</literal>.
+        <para>Into this export directory, the cluster mounts an
+            <literal>ext4</literal> file system from the DRBD device
+         <filename>/dev/drbd2</filename>.
+         This DRBD device sits on top of an LVM logical volume named
+         <literal>/dev/nfs/share</literal>.
         </para>
     </listitem>
+    <listitem>
+     <para>
+      The DRBD device <literal>/dev/drbd1</literal> is used to share the
+      NFS client states from <filename>/var/lib/nfs</filename>. This DRBD device
+      sits on top of an LVM logical volume named <literal>/dev/nfs/state</literal>.
+     </para>
+    </listitem>
   </itemizedlist>
 
   <sect2 xml:id="sec-ha-quick-nfs-resources-drbd">
-   <title>DRBD primitive and promotable clone resource</title>
+   <title>Creating DRBD primitive and promotable clone resources</title>
    <para>
-    To configure these resources, run the following commands from the
-   &crmshell;:
-   </para>
-<screen>&prompt.crm;<command>configure</command>
-&prompt.crm.conf;<command>primitive drbd_nfs \
-  ocf:linbit:drbd \
-    params drbd_resource="nfs" \
-  op monitor interval="15" role="Promoted" \
-  op monitor interval="30" role="Unpromoted"</command>
-&prompt.crm.conf;<command>clone clone-drbd_nfs drbd_nfs \
-  meta promotable="true" promoted-max="1" promoted-node-max="1" clone-max="2" \
-  clone-node-max="1" notify="true"</command>
-&prompt.crm.conf;<command>commit</command></screen>
-   <para>
-    This will create a Pacemaker promotable clone resource corresponding to the
-    DRBD resource <literal>nfs</literal>. Pacemaker should now activate your
-    DRBD resource on both nodes and promote it to the primary role on one of
-    them.
+    Create a cluster resource to manage the DRBD devices, and a promotable
+    clone to allow this resource to run on both nodes.
    </para>
+   <procedure>
+    <title>Creating a DRBD resource for NFS</title>
+    <step>
+     <para>
+      Start the <command>crm</command> interactive shell:
+     </para>
+<screen>&prompt.root;<command>crm configure</command></screen>
+    </step>
+    <step>
+     <para>
+      Create a primitive for the DRBD configuration <literal>nfs</literal>:
+     </para>
+<screen>&prompt.crm.conf;<command>primitive drbd_nfs ocf:linbit:drbd \
+  params drbd_resource="nfs" \
+  op monitor interval=15 role=Promoted timeout=20 \
+  op monitor interval=30 role=Unpromoted timeout=20 \
+  op start timeout=240 interval=0s \
+  op promote timeout=90 interval=0s \
+  op demote timeout=90 interval=0s \
+  op stop timeout=100 interval=0s</command></screen>
```

Comment:
Is all of this required for this specific setup, or is some of it incidental? (Same question for the other resources, too).

Reply:
IIRC, they are auto-populated by crmsh, based on advertisement of each specific RA.